src/web/components/SiteTour/tourStorage.ts

```diff
@@ -0,0 +1,47 @@
+import Joyride from 'react-joyride';
+
+import config from '../../../../package.json';
+
+const { version } = config;
+
+export const TourSteps: React.ComponentProps<typeof Joyride>['steps'] = [
+  {
+    target: `.profile-dropdown-button`,
+    content: `We've moved some menu items to your profile dropdown.`,
+    disableBeacon: true,
+  },
+];
+type TourData = {
+  seenForVersions: string[];
+};
+const tourStorageKey = 'tour-data';
+
+function isValidTourData(data: unknown | TourData): data is TourData {
+  return (
+    (data as TourData).seenForVersions !== undefined &&
+    Array.isArray((data as TourData).seenForVersions)
+  );
+}
+
+function getTourData(): TourData {
+  const noData = { seenForVersions: [] };
+  const unparsed = localStorage.getItem(tourStorageKey);
```

Comment:
Local storage for now, but we might need to think about some options etc. storage for each user for things like this.